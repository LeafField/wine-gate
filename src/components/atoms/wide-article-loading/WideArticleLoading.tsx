import React, { FC } from "react";

const WideArticleLoading: FC = () => {
  return (
    <div className="@container">
      <div className="flex animate-pulse flex-col border border-gray transition-colors duration-300 @[767px]:flex-row">
        <div className="aspect-video w-full shrink-0 overflow-hidden bg-gray @[767px]:aspect-auto @[767px]:h-auto @[767px]:w-[20.8125rem]"></div>
        <div className="flex-1 space-y-4 p-2 @[767px]:px-3 @[767px]:py-5">
          <div className="h-9 w-full rounded-md bg-gray">
            <span className="sr-only">読み込み中</span>
          </div>
          <div className="h-[1.875rem] w-full rounded-md bg-gray"></div>
          <div className="h-[clamp(1.125rem,2.31vw,1.5rem)] w-full rounded-md bg-gray"></div>
          <div className="h-[clamp(1.125rem,2.31vw,1.5rem)] w-full rounded-md bg-gray"></div>
        </div>
        <div></div>
      </div>
    </div>
  );
};

export default WideArticleLoading;
