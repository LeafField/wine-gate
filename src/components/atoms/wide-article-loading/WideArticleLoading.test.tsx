import { render } from "@testing-library/react";
import WideArticleLoading from "./WideArticleLoading";

describe("WideArticleLoadingの単体テスト", () => {
  test("スナップショットテスト", () => {
    const { container } = render(<WideArticleLoading />);
    expect(container).toMatchSnapshot();
  });
});
