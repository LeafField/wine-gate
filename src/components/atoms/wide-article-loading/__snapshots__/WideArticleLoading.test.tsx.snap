// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WideArticleLoadingの単体テスト スナップショットテスト 1`] = `
<div>
  <div
    class="@container"
  >
    <div
      class="flex animate-pulse flex-col border border-gray transition-colors duration-300 @[767px]:flex-row"
    >
      <div
        class="aspect-video w-full shrink-0 overflow-hidden bg-gray @[767px]:aspect-auto @[767px]:h-auto @[767px]:w-[20.8125rem]"
      />
      <div
        class="flex-1 space-y-4 p-2 @[767px]:px-3 @[767px]:py-5"
      >
        <div
          class="h-9 w-full rounded-md bg-gray"
        >
          <span
            class="sr-only"
          >
            読み込み中
          </span>
        </div>
        <div
          class="h-[1.875rem] w-full rounded-md bg-gray"
        />
        <div
          class="h-[clamp(1.125rem,2.31vw,1.5rem)] w-full rounded-md bg-gray"
        />
        <div
          class="h-[clamp(1.125rem,2.31vw,1.5rem)] w-full rounded-md bg-gray"
        />
      </div>
      <div />
    </div>
  </div>
</div>
`;
